### DISCUSSION TITLE
How to debug OmEdit with IDE (QtCreator?)

### DISCUSSION THREAD

**Discussion by AustraleanPerson** [OpenModelica/OpenModelica#12091] [General]
Opening '.PRO' file with QtCreator does not helps.

**Comment by adeas31** [comment]
This question was asked before as well. I don't remember if it was here on the discussions or on the old forum.

Anyways, I added some instructions on how to compile/debug OMEdit with Qt Creator in #12092. Try and see if you can debug OMEdit by using the instructions given.

**Comment by AustraleanPerson** [comment]
>Try and see if you can debug OMEdit by using the instructions given.

After many attempts I still can't get QtWebkit to work. Perhaps there is a way to use it from the OmDev?
- Project ERROR: Unknown module(s) in QT: webkit webkitwidgets

Also the editor says that
- In `StringHandler.cpp` `om_format.h` file not found.

_Qt Creator 5.02 and Qt 5.12.12 MinGW 64-bit_

**Reply by adeas31** [reply]
> Project ERROR: Unknown module(s) in QT: webkit webkitwidgets

If it compiles from the makefile then your OMDev is fine. Maybe QtCreator is pointing to a wrong compiler kit.

> In StringHandler.cpp om_format.h file not found.

You need to compile once using the OM top level file. That will compile Ryu lib and copy `om_format.h` to build/include.
